### PR TITLE
Added confidence score to items

### PR DESCRIPTION
**Pull request by gonzaloarro**
*Description of changes:* Added `confidence` to `Item` class per documentation at [StartStreamTranscription_ResponseSyntax](https://docs.aws.amazon.com/transcribe/latest/dg/API_streaming_StartStreamTranscription.html#API_streaming_StartStreamTranscription_ResponseSyntax) and [Item](https://docs.aws.amazon.com/transcribe/latest/dg/API_streaming_Item.html).

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by nateprewitt** [comment]
Hi @gonzaloarro, thanks for the PR! We've taken a look over the change and everything looks great. I've added a couple minor cleanup commits to get this ready to merge. I'm going to wait for a second review from @joguSD, and then I think we'll be set.

**Review by joguSD — APPROVED** [review]

**Pull request merged**
